### PR TITLE
LPS-178427 Automate |Setup Automation EVP Project

### PR DESCRIPTION
**Pull request by heiditrindade**
[LPS-178427](https://issues.liferay.com/browse/LPS-178427)

### CODE REVIEW

**Comment by heiditrindade** [comment]
ci:test:sf

**Comment by heiditrindade** [comment]
ci:test:osb-site-initializer-evp

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHeaderAndFooter.testcase [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		else {
+			JSONGroup.deleteGroupByName(groupName = "EVP");
+		}
+	}
+
+	@description = "This is a test for LPS-178427. Verify that there is a Home, Request, Organization and Reports navigation tabs on page header"
+	@priority = 5
+	test CanNavigateToPages {
+		task ("Verify that is possible to navigate in different pages") {
+			EVPNavigator.viewPage(pageList = "Home,Request,Organization,Reports");
```
[the remark this answers is not in the document]
Comment (heiditrindade) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPHeaderAndFooter.testcase:42]: Ok! I'll change the assertion to assert the page breadcrumb.

**Comment by heiditrindade** [comment]
ci:test:osb-site-initializer-evp

**Comment by nikki-pru** [comment]
ci:forward

**Comment by nikki-pru** [comment]
ci:forward

**Pull request closed without merging**